hack/conformance.sh

```diff
@@ -12,11 +12,25 @@ fi
 export KUBECONFIG=$(cluster-up/kubeconfig.sh)
 
 teardown() {
+    rv=$?
     ./sonobuoy status --json
     ./sonobuoy logs > ${ARTIFACTS}/sonobuoy.log
     results_tarball=$(./sonobuoy retrieve)
     tar -xvzf $results_tarball plugins/e2e/results/
     cp -f $(find plugins/e2e/results/* -name "*.xml") ${ARTIFACTS}/
+
+    if [ $rv -ne 0 ]; then
+        echo "error found, exiting"
+        exit $rv
+    fi
+
+    passed=$(./sonobuoy status --json | jq  ' .plugins[] | select(."result-status" == "passed")'  | wc -l)
```

Comment:
Can we do a `count_by_status` function and will be use like `cound_by_status passed` and `count_by_status failed`

Reply:
since result-status has dash, the command need double ", adding it in a function and using $var
would need playing with escapes and so on, it will be less readable
for just single case of two lines, does it worth it ?

Reply:
fair enough